lib/storage/drbd.py

```diff
@@ -398,6 +398,19 @@ def _AssembleLocal(self, minor, backend, meta, size):
       if result.failed:
         base.ThrowError("drbd%d: can't attach local disk: %s",
                         minor, result.output)
+   
+    # syncer init only for drbd => 8.4 - in 8.4 it must be set after local, not in net
+    info = DRBD8.GetProcInfo()
+    version = info.GetVersion()
+    if version["k_minor"] >= 4:
+      sync_errors = self._SetMinorSyncParams(minor, self.params)
+      # try second time because disk config for drbd resource may be still uninitialized
+      if sync_errors:
+        time.sleep(1)
```

Comment:
It's best to avoid using `time.sleep()` for cases like this. `utils.Retry` with an appropriate timeout works much better.